xarray/core/dataset.py

```diff
@@ -6723,6 +6731,9 @@ def interpolate_na(
         """
         from xarray.core.missing import _apply_over_vars_with_dim, interp_na
 
+        if keep_attrs is None:
+            keep_attrs = _get_keep_attrs(default=False)
```

Comment:
```suggestion
            keep_attrs = _get_keep_attrs(default=True)
```

IIRC we're gradually moving in this direction

Reply:
Done! Also, I noticed ffill/bfill have no keep_attrs, maybe this is a separate PR

Reply:
(Yes and actually I think we're trying to remove `keep_attrs` gradually, so fine to include it if you've already done that, but let's not add it anywhere.)